flux_led/__main__.py

```diff
@@ -940,7 +940,30 @@ def setRgbw(self, r=None, g=None, b=None, w=None, persist=True,
                 msg.append(int(w))
             else:
                 msg.append(int(0))
-            msg.append(special)
+
+            if self.protocol == "LEDENET":
+                # LEDENET devices support two white outputs for cold and warm. We set
+                # the second one here - if we're only setting a single white value,
+                # we set the second output to be the same as the first
+                if w2 is not None:
+                    msg.append(int(w2))
+                elif w is not None:
+                    msg.append(int(w))
+                else:
+                    msg.append(0)
+
+            # write mask, default to writing color and whites simultaneously
+            write_mask = 0x00
+            if self.rgbwprotocol:
```

Comment:
If I remove this and always let it calculate the write_mask, I think that fixes the remaining test failures. The question is what's the right thing to do. Hoping @mjg59 can provide more insight into what the correct thing to do here is. I can confirm it works for my controller, but want to make sure it still works for others.